build-system/tasks/make-extension/template/bento/extensions/amp-__component_name_hyphenated__/__component_version__/test/test-component.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright __current_year__ The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '#preact';
+import {__component_name_pascalcase__} from '../component';
+import {mount} from 'enzyme';
+import {waitFor} from '#testing/test-helper';
+
+describes.sandboxed('__component_name_pascalcase__ preact component v1.0', {}, (env) => {
```

Comment:
Consider adding the `// __do_not_submit__` (iirc) to ensure that this generated test can't ever be checked in